src/profile.c

```diff
@@ -1532,7 +1532,7 @@ const idclass_t profile_transcode_class =
     {
       .type     = PT_STR,
       .id       = "vcodec",
-      .name     = N_("Video CODEC"),
+      .name     = N_("Video codec"),
```

Comment:
codec vs CODEC... hmmm... it's not really an acronym... I'd have to check. "Codec" is probably correct.